runtime/binding-mqtt/src/test/java/io/aklivity/zilla/runtime/binding/mqtt/internal/config/MqttOptionsConfigAdapterTest.java

```diff
@@ -103,20 +104,20 @@ public void shouldReadOptions()
         assertThat(topic.content, instanceOf(TestValidatorConfig.class));
         assertThat(topic.content.type, equalTo("test"));
 
-        int version1 = options.versions.get(0);
-        assertThat(version1, equalTo(4));
-        int version2 = options.versions.get(1);
-        assertThat(version2, equalTo(5));
+        MqttVersion version1 = options.versions.get(0);
+        assertThat(version1.versionNumber(), equalTo(4));
+        MqttVersion version2 = options.versions.get(1);
+        assertThat(version2.versionNumber(), equalTo(5));
```

Comment:
This doesn't seem correct. The `protocolVersion` (now `protocol`) should be the wire protocol, whereas the `versionNumber` (now `specification`) should be the spec version, i.e. v3.1.1, v5.

This seems to have mixed `versionNumber` with wire protocol version 4 and 5.

Also, these asserts can compare to `MqttVersion` enums directly, instead of attributes of those enums.

If we need a test to verify the attributes, then please add `MqttVersionTest` and let's do it once for the `MqttVersion` enum constants.